meet_balls_app/tests.py

```diff
@@ -28,7 +28,7 @@ def test_display_game_event_form(self, client):
 @pytest.mark.django_db
 class Testlogin:
     def test_successful_login(self, client, player):
-        username = "daniel"
+        username = "Player"
```

Comment:
I see you didn't forget to change 'daniel' to 'Player' here well done!. 
I think if you already changed it at the `conftest.py` file, there is another test called 
`test_unsuccessful_login` where we use 
`@pytest.mark.parametrize('username, password', [
        ('wronguser', 'wrongpass'),
        ('daniel', 'wrongpass'),
        ('wronguser', 'password'),
    ])`. 
    please consider changing 'daniel' to 'Player' as well so the test will make sense (the case of correct username and wrong password).